e2e/test/MessageSendUtil.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    public static class MessageSendUtil
```

Comment:
Please avoid naming things like "Util" and making them static - it promotes bad practices such as breaking the single responsibility principle (this case).
I this case, the class appears to be a `Message` factory mixed with a portion of a test.